docs/migration/graphiql-2.0.0.md

```diff
@@ -103,7 +103,6 @@ easy access to its props, state and methods by attaching a ref to it like so:
 ```jsx
 import { createGraphiQLFetcher } from '@graphiql/toolkit';
 import { GraphiQL } from 'graphiql';
-import React from 'react';
```

Comment:
Why this change?

Reply:
good question... i need to undo some accidental bulk find replace changes that didn't trigger static analysis or test issues